apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPageTabs/EditPageTabs.tsx

```diff
@@ -163,12 +222,16 @@ export const EditPageTabs = ({ tabs, active, setActive, onAddSuccess }: Props) =
                             <Button type="button" onClick={() => handleUpdateTab()}>
                                 Save
                             </Button>
-                            <ModalToggleButton modalRef={modalRef} outline onClick={() => setIsEditing(false)} closer>
+                            <ModalToggleButton modalRef={modalRef} outline onClick={() => resetEditPageTabs()} closer>
                                 Cancel
                             </ModalToggleButton>
                         </>
                     ) : (
-                        <ModalToggleButton modalRef={modalRef} closer outline>
+                        <ModalToggleButton
+                            modalRef={modalRef}
+                            onClick={() => resetEditPageTabs()}
+                            closer
+                            disabled={selectedForDelete ? true : false}>
```

Comment:
Suggestion: ternary should not be necessary here. I believe 
```
disabled={selectedForDelete}>
```

should suffice

Reply:
selectedForDelete is an object, not boolean--checking whether it exists or not.